util.py

```diff
@@ -0,0 +1,12 @@
+def convert_str_to_bool(value: str, default: bool) -> bool:
+    try:
+        if value is None:
+            return default
+        if value.lower() == 'true':
+            return True
+        elif value.lower() == 'false':
+            return False
+        else:
+            return default
+    except Exception as e:
+        return default
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider adding specific exception type</b></div>
<div id="fix">

Consider adding type hints for the `Exception` in the `except` clause to improve type safety and make the error handling more specific.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```suggestion
          else:
              return default
     except (ValueError, AttributeError) as e:
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rkj6124/test/pull/2#issuecomment-2482194696>#7f83b6</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged